### PR TITLE
Updated COPYING file to reflect current year

### PR DESCRIPTION
**Pull request by codarrenvelvindron**

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -2,7 +2,7 @@ Unless specified otherwise in individual files or directories, the
 contents of this package is covered by the following copyright, license,
 and disclaimer:
 
-Copyright 2006 - 2015 Tarsnap Backup Inc.
+Copyright 2006 - 2016 Tarsnap Backup Inc.
 All rights reserved.
```
Comment (cperciva) [inline COPYING]: Can you split this off into a separate PR?

File: tar/multitape/multitape_fsck.c [hunk 2]
```diff
@@ -165,7 +166,7 @@ phase1(uint64_t machinenum, STORAGE_D * SD, STORAGE_R * SR,
 		goto err0;
 
 	/* Allocate space for nfiles tapemetadata structures. */
-	if ((mdats = malloc(nfiles * sizeof(struct tapemetadata *))) == NULL)
+	if (IMALLOC(mdats,nfiles,struct tapemetadata *))
```
Comment (cperciva) [inline tar/multitape/multitape_fsck.c]: Bad whitespace -- there should be spaces after the commas in a function call or macro, and there should not be any end-of-line whitespace.

File: COPYING (hunk 1 above)
Comment (codarrenvelvindron) [inline COPYING]: Split the patch and corrected it here:
https://github.com/Tarsnap/tarsnap/pull/117

**Comment by codarrenvelvindron** [comment]
Split the patch and corrected it here:
#117

**Comment by cperciva** [comment]
Thanks, I've merged #117 and bumped the copyright date myself.
